beginner_source/basics/saveloadrun_tutorial.py

```diff
@@ -34,7 +34,7 @@
 # using ``load_state_dict()`` method.
 
 model = models.vgg16() # we do not specify ``weights``, i.e. create untrained model
-model.load_state_dict(torch.load('model_weights.pth'))
+model.load_state_dict(torch.load('model_weights.pth', weights_only=True))
```

Comment:
nit: since this is the main landing page for save load, a comment along these lines  would be great 

"Setting `weights_only=True` restricts the functions that can be executed during unpickling to those required to load weights, using this flag is a best practice for loading weights."